barretenberg/cpp/src/barretenberg/vm/avm/recursion/avm_recursive_verifier.cpp

```diff
@@ -82,6 +122,32 @@ AvmRecursiveVerifier_<Flavor>::AggregationObject AvmRecursiveVerifier_<Flavor>::
 
     vinfo("verified sumcheck: ", (sumcheck_verified.has_value() && sumcheck_verified.value()));
 
+    // Public columns evaluation checks
```

Comment:
A bit tricky. I thought about it but most changes in public columns will break the avm recursive verifier. Unfortunately, the whole serialization of the public inputs is not generic at the moment. So, I can move this part in the codegen but this will be half-baked.

Reply:
Created this ticket: https://github.com/AztecProtocol/aztec-packages/issues/8849